### PR TITLE
Add permissions documentation

### PR DESCRIPTION
**Pull request by igor-egorov**
Signed-off-by: Igor Egorov <igor@soramitsu.co.jp>

### Description of the Change

Add a list of permissions with their descriptions. 
Permissions are also linked with corresponding commands or queries.

### Benefits

Now the essential part of Iroha maintenance process is documented.

### Possible Drawbacks 

Missing examples. **They will be added in the following PR.**

### Testing

You can compile docs locally using the guide from https://github.com/hyperledger/iroha/blob/develop/docs/README.md
and navigate in a browser to `_build/html/maintenance/permissions.html`

### Additional Info

There is a tool that performs automatic page generation from the csv source of
https://docs.google.com/spreadsheets/d/1aKUXLrtbD5tzJV9fSZfagFcRC_PJVrPc2EkskA1VkFU/edit?usp=sharing (visible to Soramitsu employees) - 
https://repl.it/@igor_egorov/PermissionsGoogleSheetRst

### CODE REVIEW

**Comment by laSinteZ** [comment]
Maybe better to add links to glossary (instead of "role", "quorum") and replace all "a minimum number of signatures" with "quorum", etc.

File: docs/source/maintenance/permissions.rst [hunk 1]
```diff
@@ -0,0 +1,496 @@
...
+
+Related API method
+   `Add Signatory <../api/commands.html#add-signatory>`__
+
+can_remove_my_signatory
+^^^^^^^^^^^^^^^^^^^^^^^
+
+.. Hint:: This is grantable permission.
+
+Permission that allows a specified account remove public key from the another specified account.
```
Comment (neewy) [inline docs/source/maintenance/permissions.rst]: allows a specified account to

Comment (neewy) [reply docs/source/maintenance/permissions.rst]: another?

File: docs/source/maintenance/permissions.rst [hunk 2]
```diff
@@ -0,0 +1,496 @@
...
+
+Related API method
+   `Remove Signatory <../api/commands.html#remove-signatory>`__
+
+can_set_my_quorum
+^^^^^^^^^^^^^^^^^
+
+.. Hint:: This is grantable permission.
+
+Permission that allows a specified account to set quorum number for the another specified account.
```
Comment (neewy) [inline docs/source/maintenance/permissions.rst]: another?

File: docs/source/maintenance/permissions.rst [hunk 3]
```diff
@@ -0,0 +1,496 @@
...
+
+Signatory
+---------
+
+can_add_my_signatory
+^^^^^^^^^^^^^^^^^^^^
+
+.. Hint:: This is grantable permission.
+
+Permission that allows a specified account to add an extra public key to the another specified account.
```
Comment (neewy) [inline docs/source/maintenance/permissions.rst]: another

File: docs/source/maintenance/permissions.rst [hunk 4]
```diff
@@ -0,0 +1,496 @@
...
+
+Related API method
+   `Set Account Detail <../api/commands.html#set-account-detail>`__
+
+can_set_my_account_detail
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. Hint:: This is grantable permission.
+
+Permission that allows a specified account to set details for the another specified account.
```
Comment (neewy) [inline docs/source/maintenance/permissions.rst]: another

File: docs/source/maintenance/permissions.rst [hunk 5]
```diff
@@ -0,0 +1,496 @@
...
+Related API method
+   `Create Domain <../api/commands.html#create-domain>`__
+
+Grant
+-----
+
+can_grant_can_add_my_signatory
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Allows to role owners grant `can_add_my_signatory`_ permission.
```
[the remark this answers is not in the document]
Comment (igor-egorov) [reply docs/source/maintenance/permissions.rst]: "to" will be removed

**Pull request merged**